include/roaring/isadetection.h

```diff
@@ -71,6 +71,7 @@ enum croaring_instruction_set {
   CROARING_BMI1 = 0x20,
   CROARING_BMI2 = 0x40,
   CROARING_ALTIVEC = 0x80,
+  CROARING_AVX512 =0x100,
```

Comment:
Please insert a space before 0x100 to make the code look consistent.

Reply:
i will change